### DISCUSSION TITLE
Running production Blitz with oAuth + Nginx SSL proxy

### DISCUSSION THREAD

**Discussion by PixelsCommander** [blitz-js/blitz#967] [Q&A] [answered]
I am running production Blitz on the VPS which uses Nginx to set up SSL and cache. Once I try to use Twitter auth I get "Failed to find request token in session" error. Seems cookie cant be set.

Should I set something up in a specific way to make ssl+nginx proxy work in production and let session cookies to be set?

For example, Express-session documentation says: "If you have your node.js behind a proxy and are using secure: true, you need to set "trust proxy" in express"

Found this one which seems to be not applicable to Blitz directly https://github.com/vercel/next.js/discussions/14761

**Comment by flybayer** [comment, marked as answer]
I also replied to [your GH issue](https://github.com/blitz-js/blitz/issues/966), but the the direct answer to your question here is that we haven't tested this scenario yet. So there is nothing to configure yet. Someone needs to investigate and see how to make this work.

**Reply by PixelsCommander** [reply]
Thank you for the response and sorry for the duplication. It is a bit of desperation here after 3 days of debugging. Which of these two (issue / discussion) should I remove?

**Reply by flybayer** [reply]
Yeah it's ok :). I'll remove this one.
